src/AuthorizationClient.js

```diff
@@ -1159,6 +1159,120 @@ class AuthorizationClient {
     };
   }
 
+  /**
+   * GetTenantContractId returns the ID of the tenant contract for
+   * the objectId provided from contract metadata or from fabric metadata
+   *
+   * @param libraryId
+   * @param objectId
+   * @returns {Promise<string|undefined>}
+   */
+  async GetTenantContractId(libraryId, objectId) {
```

Comment:
Move these methods to src/client/Contracts.js. AuthorizationClient is not for publicly accessible methods. See my other comment about making these methods general for all types as well.

also the client also already has methods for 'getMeta' and 'putMeta' in Contracts.js

https://eluv-io.github.io/elv-client-js/module-ElvClient_Contracts.html#.ContractMetadata
https://eluv-io.github.io/elv-client-js/module-ElvClient_Contracts.html#.MergeContractMetadata
https://eluv-io.github.io/elv-client-js/module-ElvClient_Contracts.html#.ReplaceContractMetadata